<<<<<<< HEAD
--- src/xlib/gtk.c.orig	2017-08-25 20:40:13.735005837 +1000
+++ src/xlib/gtk.c	2017-08-25 21:35:10.165215607 +1000
@@ -24,7 +24,7 @@
=======
--- src/xlib/gtk.c.orig	2018-03-26 14:52:33.000000000 +1000
+++ src/xlib/gtk.c	2018-04-02 09:59:02.201013926 +1000
@@ -24,8 +24,8 @@
>>>>>>> 31ffa0993a78202724c2a79363e4831623be7fcf
 #include <stdlib.h>
 #include <string.h>
 
-#define LIBGTK_FILENAME "libgtk-3.so"
<<<<<<< HEAD
=======
-#define LIBGTK_FILENAME_FALLBACK "libgtk-3.so.0"
>>>>>>> 31ffa0993a78202724c2a79363e4831623be7fcf
+#define LIBGTK_FILENAME "libgtk-x11-2.0.so"
+#define LIBGTK_FILENAME_FALLBACK "libgtk-x11-2.0.so.0"
 
 #define GTK_FILE_CHOOSER_ACTION_OPEN 0
 #define GTK_FILE_CHOOSER_ACTION_SAVE 1
